src/Microsoft.Health.Fhir.Api/Modules/FhirModule.cs

```diff
@@ -159,7 +158,7 @@ ResourceElement SetMetadata(Resource resource, string versionId, DateTimeOffset
                 .AsSelf()
                 .AsService<IProvideCapability>();
 
-            services.TypesInSameAssemblyAs<IProvideCapability>()
+            services.TypesInSameAssembly(typeof(IProvideCapability).Assembly, typeof(Stu3ModelInfoProvider).Assembly)
```

Comment:
Temporary, I assume

Reply:
Hopefully...I should also put these in a property / name them better. i.e. `KnownAssemblies.Core` `KnownAssemblies.CoreStu3`